lib/settings.coffee

```diff
@@ -1,26 +1,42 @@
+
```

Comment:
What do you think about renaming this file? It's a bit confusing to have both a `config` and a `settings` file.